docs/templates/docker-in-workspaces.md

```diff
@@ -109,43 +121,61 @@ resource "kubernetes_pod" "dev" {
 }
 ```
 
-> Sysbox CE (Community Edition) supports a maximum of 16 pods (workspaces) per node on Kubernetes. See the [Sysbox documentation](https://github.com/nestybox/sysbox/blob/master/docs/user-guide/install-k8s.md#limitations) for more details.
+> Sysbox CE (Community Edition) supports a maximum of 16 pods
```

Comment:
```suggestion
> Sysbox CE (Community Edition) now supports 4K pods
```
This has been increased to 4K as per https://github.com/nestybox/sysbox/releases/tag/v0.6.1

Reply:
Nice. Thanks!